### ISSUE TITLE
[BUG] ssh snapshot split line longer than 80 characters

### ISSUE THREAD

**Issue by momolemo** [netfishers-onl/Netshot#251]
Hello,

i just saw in my snapshot that some line a bit long are split onto 2 line in the snapshot result.
As a result, when i want restore the snapshot the \n added by netshot on this very long line break the restore because the switch cannot understand the 2 line as 2 commands.

I search where can be the problem and i think the problem is with the jsch channel. By default as described in the rfc4254 the terminal width, characters is set to 80. so each line longer than 80 will be split.

Do you think we can increase this width or height by config entries in netshot config file

the code impacted will be here :
https://github.com/netfishers-onl/Netshot/blob/c9a261687b875b6060707813b9a234371ed0be75/src/main/java/onl/netfishers/netshot/device/access/Ssh.java#L330

We can add something like : 
```
width = Netshot.getConfig("netshot.ssh.terminal.width", 80);
height = Netshot.getConfig("netshot.ssh.terminal.height", 24);

if (width != 80 or height != 24) {
    channel.setPtySize(width,height,width*8, height*8);
}
```

with change like this we don't change the default behavior for people not impacted by this problem.

This change is pure theory and not tested yet. I lost my compilation computer and i don't know when i can make a new one...

regards

**Comment by SCadilhac** [comment]
Thanks for the suggestion. I preferred to add this setting as a driver property as it seems to me that adjusting the pseudo terminal settings might be needed for specific drivers only.
I've implemented it in f166d3d9d626b63c694b9b0aedc434ef92246116
In the driver, you can do this:

```js
var CLI = {
  ssh: {
    config: {
      terminal: {
        pty: true,
        cols: 100,
      }
    },
    macros: {
...
```

**Comment by momolemo** [comment]
Hello,

Very good idea to put this in the driver. i did not think about it but very usefull.

Thanks for your work.
I will try this as soon as you publish a new version :)

regards
